### PR TITLE
FDG-5333 Add Hero Image Section

### PR DESCRIPTION
**Pull request by wyattch**
Ticket: https://federal-spending-transparency.atlassian.net/browse/FDG-5333
Coverage: 89.14%

### CODE REVIEW

File: src/layouts/explainer/heros/national-deficit/national-deficit-hero.tsx [hunk 1]
```diff
@@ -1,32 +1,131 @@
...
-          Compared to the same period last year (Oct YYYY (prior FY) - Mon YYYY (current month and
-          prior FY)), our national deficit has increased/decreased.
+          Compared to the same period last year
+          (Oct {previousFiscalYear} - {currentRecordMonth} {previousCalendarYear}),
+          our national deficit has {deficitStatus}.
         </p>
       </div>
+      <div className={deficitBoxContainer}>
+        <div className={deficitBox}>
+          ${displayedDeficitValue}
```
Comment (tompurkaple) [inline src/layouts/explainer/heros/national-deficit/national-deficit-hero.tsx:108]: This needs to be the value for the period of months from the prior fiscal year rather the figure from the latest fiscal year.

**Review by tompurkaple — CHANGES_REQUESTED** [review]
Changes look generally good, but prior year value is not displayed as expected.

File: src/layouts/explainer/heros/national-deficit/national-deficit-hero.tsx (hunk 1 above)
Comment (wyattch) [reply src/layouts/explainer/heros/national-deficit/national-deficit-hero.tsx:108]: Changed

**Review by tompurkaple — APPROVED** [review]
Builds and passes tests locally. Code looks good.

**Pull request merged**